src/Raven.Client/Documents/Subscriptions/SubscriptionBatch.cs

```diff
@@ -168,8 +169,11 @@ public SubscriptionBatch(RequestExecutor requestExecutor, IDocumentStore store,
         internal string Initialize(BatchFromServer batch)
         {
             _includes = batch.Includes;
+
             Items.Capacity = Math.Max(Items.Capacity, batch.Messages.Count);
             Items.Clear();
+
+            var revision = typeof(T).IsConstructedGenericType && typeof(T).GetGenericTypeDefinition() == typeof(Revision<>);
```

Comment:
I think it won't work in case of T as `dynamic`

Reply:
can you give an example?